savoirfairelinux/ringme.js
# Add eslint rules to project as suggested in #30

## Fixes #30 by @ventilooo

### Changes proposed in this pull request:

* Added the .eslintrc.json configuration file
* Modified the CONTRIBUTING.md file with instructions for eslint
* linted src/ringme.js so it conforms to all coding standards with
exception to the modules.export line
* Added a note to CONTRIBUTING.md suggesting that eslint be
installed as a global npm package

### Status


### How to verify this change

1. Follow the instructions in CONTRIBUTING.md to configure eslint
on your development machine
2. Run eslint with the command
```
eslint **/*.js
```
3. Verify that ./src/ringme.js has three linting errors
```
  77:7  error  Unexpected console statement  no-console
  284:1  error  Unexpected 'todo' comment     no-warning-comments
  285:1  error  'module' is not defined       no-undef
```

### Additional notes

Based on the maintainer's preference for this project, I recommend the following further tasks
* In package.json, add a lint command
* Disable the no-console rule in .eslintrc.json
* Temporarily disable the no-warning-comments rule until project is out of alpha
